src/testup/test_runner.rb

```diff
@@ -42,6 +42,8 @@ def run(tests, options = {})
     # @param [Hash] options
     # return [nil]
     def run_tests(tests, options)
+      options[:verbose] = TestUp.settings[:verbose_console_tests]
```

Comment:
Was this never working? Or was this recently broken?

Reply:
Well, I just tried TestUp 2.3.2 with SU 2017, and it didn't work.

Reply:
hm... it seems that `verbose_console_tests` are unused... 🤔

But, it was almost used here:
https://github.com/SketchUp/testup-2/blob/f26471dd7319d0a7f6480adde4ea8fc359de3b3f/src/testup/test_runner.rb#L73-L74

I don't recall the history of this. I need to look closer.

Reply:
The naming is odd.

One might consider `settings` to refer to TestUp, and `options` refers to Minitest?

Reply:
You might be over thinking this.  It's very possible many users either didn't use console output, or were unaware of the Minitest `--verbose`/`-v` option.  Hence, no one tried it?

Reply:
This line should be:

```rb
options[:verbose] = TestUp.settings[:verbose_console_tests] unless options.key?(:verbose)
```

Otherwise the TestUp preferences would override any CI config options passed in.

Reply:
Actually, using Settings here isn't the right place I think. That's more of an app (UI) level logic.

I think this should be passed in from `ui/runner.rb` in `event_run_tests` and `event_rerun_tests`.

Reply:
I'm not sure what should have priority, nor have I used `event_rerun_tests` recently.  I'll close and let you finish it.

```ruby
options[:verbose] = TestUp.settings[:verbose_console_tests] unless options.key?(:verbose)
```
Not sure what defaults are, maybe:
```ruby
options[:verbose] ||= TestUp.settings[:verbose_console_tests]
```